certz/certz.proto

```diff
@@ -453,11 +453,52 @@ message Certificate {
   // Actual certificate.
   // The exact encoding depends upon the type of certificate.
   // for X509, this should be a PEM encoded Certificate.
-  bytes certificate = 3;
+  // Deprecated in favour of certificate_type
+  bytes certificate = 3 [deprecated=true];
 
   // Optional.
   // Not needed if the device knows the private key or does not have to know it.
-  bytes private_key = 4;
+  // Deprecated in favour of private_key_type
+  bytes private_key = 4 [deprecated=true];
+
+  enum CertSource {
+    CERT_SOURCE_UNSPECIFIED = 0;
+
+    // certificates stored in TPM
+    CERT_SOURCE_OIDEVID_TPM = 1;
```

Comment:
the certs don't have to be stored in tpm so i would suggest just 

CERT_SOURCE_<NAME>

Reply:
ack, removed `_TPM` suffix from `CERT_SOURCE_OIDEVID_TPM` and `CERT_SOURCE_IDEVID_TPM` in latest patch